src/components/mdx-components/component-links.tsx

```diff
@@ -1,144 +1,145 @@
-  theme?: { componentName: string }
-  github?: { url?: string; package?: string }
-  npm?: { package: string }
-  storybook?: { url: string }
-  video?: { url: string }
-}
-function ComponentLinks(props: ComponentLinksProps) {
-  const { theme, github, npm, storybook, video, ...rest } = props
-  const iconColor = useColorModeValue('gray.600', 'inherit')
-
-  const githubRepoUrl = 'https://github.com/chakra-ui/chakra-ui'
-
-  const githubLink = (github?.url || github?.package) && (
-    <WrapItem>
-      <ComponentLink
-        url={
-          github.url || `${githubRepoUrl}/tree/main/packages/${github.package}`
-        }
-        icon={FaGithub}
-        iconColor={iconColor}
-        iconSize='1rem'
-      >
-        {t('component.mdx-components.component-links.view-source')}
-      </ComponentLink>
-    </WrapItem>
-  )
-
-  const npmLink = npm?.package && (
-    <WrapItem>
-      <ComponentLink
-        url={`https://www.npmjs.com/package/${npm.package}`}
-        icon={FaNpm}
-        iconSize='2rem'
-        iconColor='red.500'
-      >
-        {npm.package}
-      </ComponentLink>
-    </WrapItem>
-  )
-
-  const storybookLink = storybook?.url && (
-    <WrapItem>
-      <ComponentLink
-        url={storybook.url}
-        icon={StorybookIcon}
-        iconSize='1.25rem'
-        iconColor='pink.500'
-      >
-        {t('component.mdx-components.component-links.view-storybook')}
-      </ComponentLink>
-    </WrapItem>
-  )
-
-  const videoLink = video?.url && (
-    <WrapItem>
-      <ComponentLink
-        url={video.url}
-        icon={FaYoutube}
-        iconSize='1.2rem'
-        iconColor='red.500'
-      >
-        {t('component.mdx-components.component-links.view-video')}
-      </ComponentLink>
-    </WrapItem>
-  )
-
-  const themeComponentLink = theme && (
-    <WrapItem>
-      <ComponentLink
-        url={`${githubRepoUrl}/tree/main/packages/theme/src/components/${theme.componentName}.ts`}
-        icon={FaGithub}
-        iconColor={iconColor}
-        iconSize='1rem'
-      >
-        {t('component.mdx-components.component-links.view-theme-source')}
-      </ComponentLink>
-    </WrapItem>
-  )
-
-  return (
-    <Wrap className='component-links' mt='2rem' spacing='4' {...rest}>
-      {githubLink}
-      {themeComponentLink}
-      {npmLink}
-      {storybookLink}
-      {videoLink}
-    </Wrap>
-  )
-}
-
-export default ComponentLinks
+import {
+  HStack,
+  Icon,
+  Link,
+  useColorModeValue,
+  ButtonGroup,
+  Button,
+  ButtonProps,
+  Text,
+} from '@chakra-ui/react'
+import React from 'react'
+import { FaGithub, FaNpm, FaYoutube } from 'react-icons/fa'
+import StorybookIcon from '../storybook-icon'
+import { t } from 'utils/i18n'
+
+type ComponentLinkProps = ButtonProps & {
+  icon: React.ElementType
+  url: string
+  iconSize?: string
+  iconColor?: string
+}
+
+function ComponentLink(props: ComponentLinkProps) {
+  const { icon, url, children, iconSize, iconColor, ...rest } = props
+  return (
+    <Button

```

Comment:
It would be nice if the buttons could match the old design ? What do you think?

Reply:
Sure! I personally favor retaining the touch space the `Button` provides for the mobile accessibility. There is a compromise here. 😄 
![image](https://user-images.githubusercontent.com/65234762/169775176-53edf723-b000-4bb1-91a1-f1a317125069.png)

Reply:
A compromise is what I meant. Was just missing the border and backgroundcolor when I checked :)